### DISCUSSION TITLE
rpm package

### DISCUSSION THREAD

**Discussion by jtagcat** [espanso/espanso#1208] [Feature requests and Ideas]
For Fedora, OpenSUSE, and other friends.

**Comment by federico-terzi** [comment]
Hey @jtagcat,

Thank you for the feedback! That might be a good idea indeed, although I don't have any experience with that. If anyone from the community is willing to become the maintainer, I'll be happy to collaborate!

Cheers :)

**Comment by rickykresslein** [comment]
I just created a repository and RPM for openSUSE. You can check it out [here](https://github.com/federico-terzi/espanso/issues/603).

**Comment by lobau** [comment]
+1 would love to be able to install espanso on Fedora 34!

**Comment by pointydev** [comment]
In lieu of acceptance into the official Fedora repos, a [Copr](https://copr.fedorainfracloud.org/) repo may be able to accommodate users' desires in the meantime as they are fairly [simple to enable](https://docs.pagure.org/copr.copr/how_to_enable_repo.html). Documentation for setting them up can be found [here](https://docs.pagure.org/copr.copr/user_documentation.html).

**Comment by ghost** [comment]
It's a pity that this is still stuck. Since wayland is now default for more and more distros, the need to compile (except for debian-ubuntu) is really undesirable and an obstacle for many users.

**Comment by xplosionmind** [comment]
Hi! Any news about this?  
I have never packaged anything, but maybe this could be the right occasion to start. I just need some guidance…

**Reply by AucaCoyan** [reply]
Hello! @xplosionmind so far no news yet. [The current installation methods](https://espanso.org/docs/install/linux/#choosing-the-right-install-method) only include .deb, appimage snap or manual compilation (which is a pain in my poor dev experience)
I can't help you because I've never did a rpm package, but I can say that we have a discord that is taking off with few members. Maybe we can write and have some help there!
https://discord.gg/DFcCNDg7bB
see you there

**Reply by xplosionmind** [reply]
As reported in #1791, I am stuck because binary compilation is failing for me…

**Comment by AucaCoyan** [comment]
Closing in favour of the meta issue #1948
